packages/terra-status-view/tests/wdio/status-view-spec.js

```diff
@@ -0,0 +1,189 @@
+/* global browser, Terra, before */
+const shouldTheme = (customProperties) => {
+  Object.entries(customProperties).forEach(([key, value]) => {
+    it(`themed [${key}]`, () => {
+      browser.setCSSCustomProps({ [key]: value });
+      expect(browser.checkElement('[data-reactroot]')).to.matchReference();
+    });
+  });
+};
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
```

Comment:
We dont' need to test all these viewports, if we need responsive testing I'd do a few things:

1. Have an explicit responsive test
2. Do edge testing (recommended). Example, do tiny and medium breakpoints to verify component flexes well.